source/getting-started/core-concepts.md

```diff
@@ -59,3 +59,25 @@ While templates describe how a user interface looks, components control how the
 Components consist of two parts: a template written in Handlebars, and a source file written in JavaScript that defines the component's behavior. For example, our property rental application might have a component for displaying all the rentals called `all-rentals`, and another component for displaying an individual rental called `rental-tile`. The `rental-tile` component might define a behavior that lets the user hide and show the image property of the rental.
 
 Let's see these core concepts in action by building a property rental application in the next lesson.
+
+## Hooks
+
+In Ember, we use the term **hook** for functions that are automatically called within the Ember application. These are functions that can be expected to be called automatically, rather than having to call them specifically.
```

Comment:
Could you call them methods instead of functions? It is splitting hairs a bit, but I think it'll be more consistent with the rest of the Guides, and for people that know the difference, they'll know that the hooks are bound.